src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -486,6 +514,7 @@ def _unpack_build(self,
                     http_build_url=None):
     """Unpacks a build from a build url into the build directory."""
     # Track time taken to unpack builds so that it doesn't silently regress.
+    self._emmit_build_age_metric(build_url)
```

Comment:
is this correct when we already have line 906?

Reply:
Line 906 only applies for _CustomBuild_, that overrides **setup()** and does not call **_unpack_build()** from the _Build_ class.

All other classes end up calling **_unpack_build()** from the _Build_ class, more specifically through the **setup()** method inherited from _RegularBuild_ (with the exception of _SymbolizedBuild_, that overrides **setup()** but ends up calling the **_unpack_build()** method in the _Build_ class anyway). 

So this does look solid to me.

Reply:
Ack, I misread this. My bad!